### PR TITLE
SAK-30573: Increase space beneath 'View' label and dropdown

### PR DESCRIPTION
**Pull request by nicholaswilson100**
Adding an extra table/row is how padding is done elsewhere on the page.

### CODE REVIEW

File: signup/tool/src/webapp/signup/signupMeetings.jsp [hunk 1]
```diff
@@ -227,7 +227,11 @@
 						<h:outputText value="&nbsp;" escape="false" rendered="#{!SignupMeetingsBean.enableExpandOption}"/>
 					</h:panelGroup>
 				</h:panelGrid>
-				
+
+				<h:panelGrid columns="1">
+					<h:outputText value="&nbsp;" escape="false"/>
```
Comment (bjones86) [inline signup/tool/src/webapp/signup/signupMeetings.jsp]: I think this should be accomlished via CSS rather than inserting non-breaking spaces.

**Comment by nicholaswilson100** [comment]
Yes that is the better thing to do.

**Comment by nicholaswilson100** [comment]
Thanks Brian

**Comment by bjones86** [comment]
LGTM, thanks @nicholaswilson100

**Pull request merged**